http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/HeadersIT.java

```diff
@@ -93,6 +95,24 @@ void testPathSpecificHeaderRulesOrder() {
         cacheControlMatches(response, "max-age=1");
     }
 
+    /*
+     * This test check 3.8.6 backport https://github.com/quarkusio/quarkus/pull/41411 will fails with quarkus-resteasy
+     * HeadersMessageBodyWriter just works with quarkus-rest and not with quarkus-resteasy
+     */
+    @Test
+    @Tag("https://github.com/quarkusio/quarkus/pull/41411")
+    void testWithNoAcceptHeader() {
+        Header header = new Header("Accept", null);
+        given()
+                .when()
+                .header(header)
+                .get("/headers/no-accept")
+                .then()
+                .statusCode(200)
+                .body(not("Headers response: ok"))
```

Comment:
As you created bug shoudn't we just check if  `.body(is("Headers response: ok"))` and disable the test with the link to your upstream issue. 

If it's really bug and will be fixed, this will break in daily run.

Reply:
Yes, u are right.